src/org/ods/util/Project.groovy

```diff
@@ -507,7 +507,7 @@ class Project {
     }
 
     List<Map> getDocumentTrackingIssuesNotDone(List<String> labels) {
-        return this.getDocumentTrackingIssues(labels).findAll { !it.status.equalsIgnoreCase("done") }
+        return this.getDocumentTrackingIssues(labels).findAll { !it.status?.equalsIgnoreCase("done") }
```

Comment:
@hrcornejo did you see issues where the status field was `null`? If yes, that's an error that should be fixed in the data source / data extraction, and preferrably not here. Using Groovy`s _safe navigation operator_ `?.` too often would make me worry about the consistency of our data. Thoughts?

Reply:
This method is not used.